packages/react-vapor/src/components/input/tests/Label.spec.tsx

```diff
@@ -13,48 +13,29 @@ describe('Label', () => {
     });
 
     describe('<Label />', () => {
-        let label: ReactWrapper<ILabelProps, any>;
-
-        beforeEach(() => {
-            label = mount(<Label />, {attachTo: document.getElementById('App')});
-        });
-
-        afterEach(() => {
-            label.detach();
-        });
-
         it('should set inner label classes when specified', () => {
             const innerLabelClass = 'valid';
-            const classes = [innerLabelClass];
-            expect(label.find('label').hasClass(innerLabelClass)).toBe(false);
-
-            label
-                .setProps({classes})
-                .mount()
-                .update();
+            const classesTest = [innerLabelClass];
+            const label = shallow(<Label classes={classesTest} />);
             expect(label.find('label').hasClass(innerLabelClass)).toBe(true);
         });
 
         it('should set inner label valid message when specified', () => {
             const message = 'salut';
-            expect(label.find('label').prop('data-valid-message')).toBe(undefined);
-
-            label
-                .setProps({validMessage: message})
-                .mount()
-                .update();
+            const label = shallow(<Label data-valid-message={message} />);
             expect(label.find('label').prop('data-valid-message')).toBe(message);
         });
 
         it('should set inner label invalid message when specified', () => {
             const message = 'salut';
-            expect(label.find('label').prop('data-invalid-message')).toBe(undefined);
-
-            label
-                .setProps({invalidMessage: message})
-                .mount()
-                .update();
+            const label = shallow(<Label data-invalid-message={message} />);
             expect(label.find('label').prop('data-invalid-message')).toBe(message);
         });
+
+        it('should set inner label data nightwatch selector when specified', () => {

```

Comment:
I think it would be best not to mention nightwatch here.

```suggestion
        it('should pass down any other attribute to the underlying label tag', () => {
```